src/main/java/appeng/core/sync/packets/AbstractPacket.java

```diff
@@ -0,0 +1,43 @@
+//
+//package appeng.core.sync.packets;
+//
+//
+//import io.netty.buffer.ByteBuf;
+//
+//import appeng.core.sync.AppEngPacket;
+//
+//
+///**
+// * An abstract packet providing basic information about an {@link AppEngPacket}.
+// *
+// */
+//public abstract class AbstractPacket implements AppEngPacket
+//{
+//
+//	private int writtenBytes = 0;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)